src/lib/containers/entity_finder/EntityFinder.tsx

```diff
@@ -314,74 +272,92 @@ export const EntityFinder: React.FunctionComponent<EntityFinderProps> = ({
-                      windowResizeAware
-                    >
-                      <ReflexElement minSize={200} size={350}>
-                        <TreeView
-                          sessionToken={sessionToken}
-                          setDetailsViewConfiguration={setConfigFromTreeView}
-                          showDropdown={true}
-                          visibleTypes={visibleTypesInTree}
-                          initialScope={initialScope}
-                          initialContainerId={initialContainerId}
-                        />
-                      </ReflexElement>
-                      <ReflexSplitter></ReflexSplitter>
-                      <ReflexElement minSize={400}>
-                        <EntityDetailsList
-                          sessionToken={sessionToken}
-                          configuration={configFromTreeView}
-                          showVersionSelection={showVersionSelection}
-                          selected={selectedEntities}
-                          includeTypes={showTypes}
-                          selectableTypes={selectableTypes}
-                          selectColumnType={
-                            selectMultiple ? 'checkbox' : 'radio'
-                          }
-                          toggleSelection={toggleSelection}
-                        />
-                      </ReflexElement>
-                    </ReflexContainer>
-                  )}
-                </SizeMe>
-              </div>
+              {treeOnly ? (
+                <div style={{ width: '550px', margin: 'auto' }}>
+                  <TreeView
+                    sessionToken={sessionToken}
+                    toggleSelection={toggleSelection}
+                    setDetailsViewConfiguration={() => {}}
+                    showDropdown={true}
+                    visibleTypes={visibleTypesInTree}
+                    initialScope={initialScope}
+                    projectId={projectId}
+                    initialContainer={initialContainer}
+                    showFakeRoot={false}
+                    nodeAppearance={NodeAppearance.SELECT}
+                    setBreadcrumbItems={() => {}}
+                    selectableTypes={selectableTypes}
+                  />
+                </div>
+              ) : (
+                <div className="EntityViewReflexContainer">
+                  <SizeMe>
+                    {({ size }) => (
+                      <ReflexContainer
+                        key={(!!size.width).toString()}
+                        orientation="vertical"
+                        windowResizeAware
+                      >
+                        <ReflexElement
+                          className="TreeViewReflexElement"
+                          minSize={200}
+                          size={275}
+                        >
+                          <TreeView
+                            sessionToken={sessionToken}
+                            toggleSelection={() => {}}
```

Comment:
In the dual-pane view, the tree does not control the selection, only the configuration